test/unit/lib/otel/trace-propagator.test.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const assert = require('node:assert')
+const test = require('node:test')
+const helper = require('../../../lib/agent_helper')
+const otel = require('@opentelemetry/api')
+const NewRelicTracePropagator = require('../../../../lib/otel/trace-propagator')
```

Comment:
If you like, you can:
```suggestion
const NewRelicTracePropagator = require('#agentlib/otel/trace-propagator.js')
```

Reply:
forgot about this, ill start using it in tests